XRTK-Core/Packages/com.xrtk.core/Editor/PackageInstaller.cs

```diff
@@ -119,7 +123,6 @@ private static string CopyAsset(this string rootPath, string sourceAssetPath, st
                 Directory.CreateDirectory(Directory.GetParent(destinationPath).FullName);
 
                 File.Copy(sourceAssetPath, destinationPath);
-                File.Copy($"{sourceAssetPath}{META_SUFFIX}", $"{destinationPath}{META_SUFFIX}");
```

Comment:
We should def be copying the meta files too.

Reply:
Please read the full scope of the change before commenting.  .meta is one of the file types ALREADY included.  I copied the method used to regenerate GUIDs on your suggestion and that already included Meta files.

Check in the UnityFileHelper that lists all the file types included